playground/orleans/OrleansAppHost/Program.cs

```diff
@@ -23,7 +23,7 @@
 // to test end developer dashboard launch experience. Refer to Directory.Build.props
 // for the path to the dashboard binary (defaults to the Aspire.Dashboard bin output
 // in the artifacts dir).
-builder.AddProject<Projects.Aspire_Dashboard>(KnownResourceNames.AspireDashboard);
+builder.AddProject<Projects.Aspire_Dashboard>(KnownResourceNames.AspireDashboard).ExcludeFromManifest();
```

Comment:
This should be special logic applied to the known resource name.

Reply:
I used to do this when I had the lifecycle hook but we moved the logic. I'll add a new lifecycle hook back in. We can't exclude from the manifest where we moved it to (it's too late).

Reply:
Just add logic to the manifest generator to leave out this resource. It doesn't need to be a hook.

Reply:
Maybe lower tech is just shared code.. in these playground projects that do this...

Reply:
Shared code doesn't help because you still have to invoke it from somewhere. I'll add some exclusion logic to the manifest generator. Although TBH its just a hack, the better approach is a hook.

Reply:
Shared code works, all of the playground samlpes would include this helper extension method `AddDashboard()`. That would do the above.